ASM/c/message.c

```diff
@@ -237,9 +237,7 @@ uint8_t treasure_chest_game_show_message = 0;
 // Displays a warning message if the player attempted the Treasure Chest Game without
 // Lens of Truth when settings require it.
 void treasure_chest_game_message() {
-    if (z64_file.items[Z64_SLOT_LENS] != Z64_ITEM_LENS) {
```

Comment:
Checking for Lens here wasn't strictly required, but it added context and some defense-in-depth if this got misused. If this is changed to just always set the variable to 1, it makes more sense to replace the jump in `chest_game.asm` to write to the byte directly.

Reply:
That's fair. Added back.